core/src/test/java/org/arquillian/smart/testing/custom/assertions/ConfigurationAssert.java

```diff
@@ -0,0 +1,75 @@
+package org.arquillian.smart.testing.custom.assertions;
+
+import org.arquillian.smart.testing.RunMode;
+import org.arquillian.smart.testing.configuration.Configuration;
+import org.arquillian.smart.testing.configuration.Range;
+import org.assertj.core.api.AbstractAssert;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class ConfigurationAssert extends AbstractAssert<ConfigurationAssert, Configuration> {
+
+    ConfigurationAssert(Configuration actual) {
+        super(actual, ConfigurationAssert.class);
+    }
+
+    public static ConfigurationAssert assertThat(Configuration config) {
+        return new ConfigurationAssert(config);
+    }
+
+    public ConfigurationAssert hasMode(RunMode mode) {
+        isNotNull();
+
+        if (!Objects.equals(actual.getMode(), mode)) {
+            failWithMessage("Expected run mode to be <%s> but was <%s>", mode, actual.getMode());
+        }
+        return this;
+    }
+
+    public ConfigurationAssert isAppliedTo(String applyTo) {
+        isNotNull();
+
+        if (!Objects.equals(actual.getApplyTo(), applyTo)) {
+            failWithMessage("Expected plugin to be <%s> but was <%s>", applyTo, actual.getApplyTo());
+        }
+        return this;
+    }
+
+    public ConfigurationAssert hasAppliedStrategies(String[] strategies) {
```

Comment:
How about  `hasAppliedStrategies(String... strategies)`? 
So Instead of ` .hasAppliedStrategies(new String[] {"changed"})`, user can simply use ` .hasAppliedStrategies("changed")`